app/frontend/src/screens/Dashboard/Dashboard.jsx

```diff
@@ -56,6 +56,7 @@ const Dashboard = React.memo((props) => {
       changeSelectedValue(null);
     }
   });
+  const userIdList = [useSelector(state => lodash.get(state.users.me, "id"))];
```

Comment:
Wrap it in list in the inner most u can - get params of fetch.
Change it to user ID
The calendar never showing details of multiple users.

Reply:
I made it from here so the calendar will be generic, we want to use the calendar in different locations (commander space for example) that we don't necessarily want to get the user data but we want to get other users.

Reply:
yes i agree its here.
but instead of a list do it a simple var.
there is no need for a list

Reply:
also it is strange that the calendar needs more than 1 man if we always display one

Reply:
If I remember correctly I need to send list if users id in the get request

Reply:
In the past we talked about showing all the soldiers calendar in the commander space (all of them in one calendar

Reply:
really?

Reply:
ok then..